README.md

```diff
@@ -50,14 +50,12 @@ Please find more information in the related [wiki page](https://github.com/artiv
 
 ___
 
-<p align="center">
-  <a href="#installation">Installation</a> •
-  <a href="#features">Features</a> •
-  <a href="#documentation">Documentation</a> •
-  <a href="#tutorials-and-application-demos">Tutorials</a> •
-  <a href="#publications">Publications</a> •
-  <a href="#contributing">Contributing</a>
-</p>
+* [Installation](#installation)
```

Comment:
This changes the layout from a centered one-liner to a left-aligned list. I'd rather keep the original layout, can you revert this particular change ?

Reply:
Hm, github doesn't make it easy to accept part of a commit but not all of it, does it?

Forgive me, I'm more used to patches by email where this isn't a problem.

I'll rewrite the patch.

Reply:
You can simply push another commit if this is simpler, I'll squash them at merge.

Reply:
I'll get around to it. I wish you could just do it without my intervention. Github is dumb.